kinto/core/storage/postgresql/__init__.py

```diff
@@ -243,6 +243,9 @@ def create(self, collection_id, parent_id, record, id_generator=None,
         VALUES (:object_id, :parent_id,
                 :collection_id, (:data)::JSONB,
                 from_epoch(:last_modified))
+        ON CONFLICT (id, parent_id, collection_id) DO UPDATE
+        SET data = (:data)::JSONB,
+            last_modified = from_epoch(:last_modified)
```

Comment:
I'm not sure, but if `last_modified` is not provided and there is a conflict, then this could fail (`last_modified` can not be null, but maybe the trigger will initialize it, we have to check)